DEFINITION.md

```diff
@@ -18,6 +18,14 @@ ecosystem of open source, vendor-neutral projects. We democratize state-of-the-a
 innovations accessible for everyone.
 
 
+## Ukrainian 🇺🇦
+
+Нативні хмарні технології дозволяють організаціям створювати та запускати масштабовані додатки в сучасних динамічних середовищах, таких як публічні, приватні та гібридні хмари. Контейнери, сервісні сітки, мікросервіси, незмінна інфраструктура і декларативні API є прикладами такого підходу.
```

Comment:
> сервісні сітки

I believe, Service Meshes are meant here, I'd look for a better translation of the definition.